config/database.yml

```diff
@@ -1,25 +1,20 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
+  username: postgres
+  password: xxxxx #<-- which you have defiend while installing postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
   <<: *default
-  database: storage/development.sqlite3
+  database: RoR_Platform
```

Comment:
Normally we use a name that identifies the project and the environment, like `planit_development`

Reply:
Btw, if you change the name, first drop the database, then change the name and then you are going to have to create the database again, and run the migrations.